index.html

```diff
@@ -173,126 +173,447 @@ <h2>
+                      <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a>.
+                    </li>
+                    <li>
+                      <a>Reject</a> <a>promise</a> with a "{{InvalidStateError}}" {{DOMException}}.
+                    </li>
+                    <li>
+                      Return <a>promise</a>.
+                    </li>
+                  </ol>
+                </ol>
+              </li>
+              <li>
+                If the user agent's <a>pointer-lock target</a>'s <a>shadow-including
+                root</a> is equal to <a>this</a>'s <a>shadow-including root</a>, then:
+                <ol>
+                  <li>
+                    If <a>options</a> are equivalent to the current <a>pointer-lock options</a> then the <a>pointer-lock target</a>
+                    must be updated to this element, a <a>pointerlockchange</a>
+                    event sent, and the promise must <a>resolve</a>.
+                  </li>
+                  <li>
+                    If <a>options</a> are not equivalent to the user agent's current
+                    <a>pointer-lock options</a> and <a>options</a> are not supported:
+                    <ol>
+                      1. <a href="https://html.spec.whatwg.org/multipage/webappapis.html#queue-an-element-task">
+                      Queue an element task</a> on the 
+                      <a href="https://html.spec.whatwg.org/multipage/webappapis.html#user-interaction-task-source">
+                      user interaction task source</a>, given this, to perform the following steps:
+                      <ol type='i'>
+                        <li>
+                          <a href=https://dom.spec.whatwg.org/#concept-event-fire">Fire an event</a> named
+                          <a>pointerlockerror</a> at <a>this</a>'s 
+                          <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a>.
+                        </li>
+                        <li>
+                          <a href="https://html.spec.whatwg.org/multipage/webappapis.html#queue-an-element-task">
+                            Queue an element task</a> on the 
+                            <a href="https://html.spec.whatwg.org/multipage/webappapis.html#user-interaction-task-source">
+                            User Interaction task source</a>, given this, to perform the following steps:
+                            <ol type='i'>
+                              <li>
+                                <a href=https://dom.spec.whatwg.org/#concept-event-fire">Fire an event</a> named
+                                <a>pointerlockerror</a> at <a>this</a>'s 
+                                <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a>.
+                              </li>
+                              <li>
+                                The <a>promise</a> must be resolved.
```

Comment:
Why would you resolve the promise in an error condition (not supported options)?